turbopy/core.py

```diff
@@ -245,22 +245,31 @@ def read_diagnostics_from_input(self):
                       self.input_data["Diagnostics"].items()
                       if not Diagnostic.is_valid_name(k)}
 
-            # todo: implement a system for making default file names
             if "directory" in params:
                 d = Path(params["directory"])
-                d.mkdir(parents=True, exist_ok=True)
+            else:
+                # Set a default output directory
+                d = Path("default_output")
+                params["directory"] = d
```

Comment:
Never mind... changed my mind already. Please switch this to save the `str` rather than the `Path` object.
```suggestion
                params["directory"] = str(d)
```